tests/e2e/logging/test_openshift-logging.py

```diff
@@ -171,3 +186,58 @@ def test_create_new_project_to_verify_logging(self, create_pvc_and_deploymentcon
         pod_obj.run_io(storage_type='fs', size=6000)
 
         self.validate_project_exists(pvc_obj)
+
+    @pytest.mark.polarion_id("OCS-650")
+    @tier4
+    def test_respin_osd_pods_to_verify_logging(self, create_pvc_and_deploymentconfig_pod):
+        """
+        This function creates projects before and after respin of osd
+        and verify project existence in EFK stack.
+        1. Creates new project with PVC and app-pods
+        2. Respins osd
+        3. Logs into the EFK stack and checks for the health of cluster-logging
+        4. Logs into the EFK stack and checks project existence
+        5. Checks for the shards of the project in the EFK stack
+        6. Creates new project and checks the existence again
+        """
+
+        # Create 1st project and app_pod
+        dc_pod_obj, dc_pvc_obj = create_pvc_and_deploymentconfig_pod
+
+        project1 = dc_pvc_obj.project.namespace
+
+        # Delete the OSD pod
+        disruption = disruption_helpers.Disruptions()
+        disruption.set_resource(resource='osd')
+        disruption.delete_resource()
+
+        # Check the health of the cluster-logging
+        assert ocp_logging_obj.check_health_of_clusterlogging()
+
+        # Check for the 1st project created in EFK stack before the respin
+        self.validate_project_exists(dc_pvc_obj)
+
+        # Check the files in the project
+        elasticsearch_pod_obj = self.get_elasticsearch_pod_obj()
+
+        project1_filecount = elasticsearch_pod_obj.exec_cmd_on_pod(
+            command=f'es_util --query=project.{project1}.*/_count'
+        )
+        assert project1_filecount['_shards']['successful'] == 0, f"No files found in project {project1}"
+        logger.info(f'The files in the project 1 {project1_filecount}')
+
+        # Create another app_pod in new project
+        pod_obj, pvc_obj = create_pvc_and_deploymentconfig_pod
+
+        project2 = pvc_obj.project.namespace
+
+        # Check the 2nd project exists in the EFK stack
+        self.validate_project_exists(pvc_obj)
+
+        project2_filecount = elasticsearch_pod_obj.exec_cmd_on_pod(
+            command=f'es_util --query=project.{project2}.*/_count', out_yaml_format=True
+        )
+        logger.info(f'The files in the project2 {project2_filecount}')
+
+        assert project2_filecount['_shards']['successful'] == 0, f"No files found in project {project2}"
+        logger.info(f'The files in the project 2 {project2_filecount}')
```

Comment:
is this logging needed since you already logging the same in https://github.com/red-hat-storage/ocs-ci/pull/744/files#diff-150c594678615e7fbf901cc6855acbcbR240

Reply:
ACK, The comment will be taken in the next PR